airbyte-integrations/connectors/source-google-search-console/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7-slim
```

Comment:
we should use alpine, see source-zendesk-support for an example

Reply:
tried an image from the zendesk connector, got timezone detection error during testing, rolled back to python:3.7-slim

Reply:
TZ issue should be fixed, see latest templates